features/promotion/applying_promotion_coupon/applying_promotion_coupon_with_per_customer_usage_limit.feature

```diff
@@ -35,28 +35,38 @@ Feature: Applying promotion coupon with per customer usage limit
 
     @ui
     Scenario: Receiving no discount from valid coupon that has reached its per customer usage limit
-        Given I placed an order "#00000022"
+        Given this coupon can be used once per customer
+        And I placed an order "#00000022"
         And I bought a single "PHP T-Shirt" using "SANTA2016" coupon
         And I chose "Free" shipping method to "United States" with "Cash on Delivery" payment
-        And I placed an order "#00000023"
-        And I bought a single "PHP Socks" using "SANTA2016" coupon
-        And I chose "Free" shipping method to "United States" with "Cash on Delivery" payment
         When I add product "PHP T-Shirt" to the cart
         And I use coupon with code "SANTA2016"
         Then I should be notified that the coupon is invalid
         And my cart total should be "$100.00"
         And there should be no discount
 
     @ui
-    Scenario: Cancelled orders should not influence per customer usage limit
-        Given I placed an order "#00000022"
+    Scenario: Cancelled orders are not affecting per customer usage limit by default
```

Comment:
What about `Cancelled orders do not affect per customer usage limit by default` ?